### PR TITLE
implement probot stale

### PR DESCRIPTION
**Pull request by jiaw37**
**Task name**: implement probot stale 

**Description**:  after 1 day, close the github issue and comment to redirect user to ADO dev community.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+daysUntilStale: 1
+
+daysUntilClose: 1
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
```
Comment (edgarrs) [inline .github/stale.yml:6]: target it to only issues we own

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  In order to consolidate to fewer feedback channels, we've moved suggestions and
+  issue reporting to [Developer Community](https://developercommunity.visualstudio.com/spaces/21/index.html).
+
+limitPerRun: 30
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (edgarrs) [inline .github/stale.yml:31]: should this be true?

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: redirect-to-dev-community
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
```
Comment (edgarrs) [inline .github/stale.yml:24]: Is the character greater than on purpose or is it a typo?

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: redirect-to-dev-community
```
Comment (edgarrs) [inline .github/stale.yml:21]: shouldn't these values have quotation marks?

File: .github/stale.yml (hunk 3 above)
Comment (jiaw37) [reply .github/stale.yml:24]: No, that is how comments are made

File: .github/stale.yml (hunk 2 above)
Comment (jiaw37) [reply .github/stale.yml:31]: We already post a comment when we set the issue to stale, commenting on close might be redundant

File: .github/stale.yml (hunk 4 above)
Comment (jiaw37) [reply .github/stale.yml:21]: No, there are no quotes on the doc

File: .github/stale.yml (hunk 1 above)
Comment (jiaw37) [reply .github/stale.yml]: updated

**Review by edgarrs — APPROVED** [review]

**Pull request merged**